Tickr/TickrTests/TickrTests.swift

```diff
@@ -0,0 +1,288 @@
+//
+//  TickrTests.swift
+//  TickrTests
+//
+//  Created by Stephen Payne on 4/8/16.
+//  Copyright © 2016 Stephen Payne. All rights reserved.
+//
+
+import XCTest
+
+@testable import Tickr
+
+class TickrTests: XCTestCase {
+    let testData = TickrTestData()
+    
+    override func setUp() {
+        super.setUp()
+        //create test data for all tests to use
+        
+
+    }
+    
+    override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        super.tearDown()
+    }
+    
+    
+    
+}
+
+class StockManagerTests: TickrTests {
+    var stocks = [Stock]()
+    var manager = StockManager.sharedInstance
+    
+    
+    //MARK: - Parsing Tests
+    func testParseJSON() {
+        do {
+            let json = try NSJSONSerialization.JSONObjectWithData(
+                TickrTestData.singleResultData!,
+                options:NSJSONReadingOptions.MutableContainers
+                ) as! NSDictionary
+            
+            let stockArrayWithData = manager.parseJSON(json)
+            XCTAssertNotNil(stockArrayWithData, "Data couldn't be parsed. Check the data file to make sure the JSON structure hasn't changed")
```

Comment:
when you are testing that something was parsed, not only do you want to test that the array is non-empty, but you also want to check that the properties of the objects in the array are correct, so at least check the first object of this array